### PR TITLE
[FAN-384] adding multiple chains

### PR DESCRIPTION
**Pull request by XuMinhui**

### CODE REVIEW

File: src/modules/layout/components/Header/components/SelectChainDialog/SelectChainDialog.tsx [hunk 1]
```diff
@@ -0,0 +1,133 @@
...
+        chainId: '0x38',
+        chainName: 'Binance Smart Chain Mainnet',
+        nativeCurrency: {
+          name: 'Binance',
+          symbol: 'BNB',
+          decimals: 18,
+        },
+        rpcUrls: ['https://bsc-dataseed4.binance.org'],
+        blockExplorerUrls: ['https://bscscan.com/'],
+      },
```
Comment (Denise54321) [inline src/modules/layout/components/Header/components/SelectChainDialog/SelectChainDialog.tsx:74]: All blockChain explorer urls can be set in BlockChainExplorerAddress

File: src/modules/layout/components/Header/components/SelectChainDialog/SelectChainDialog.tsx [hunk 2]
```diff
@@ -0,0 +1,133 @@
...
+    },
+  ];
+
+  const renderCard = ({
+    icon,
+    title,
+    subTitle,
+    chainConfig,
+  }: {
+    icon: JSX.Element;
```
Comment (Denise54321) [inline src/modules/layout/components/Header/components/SelectChainDialog/SelectChainDialog.tsx]: ReactNode

File: src/modules/layout/components/Header/components/SelectChainDialog/SelectChainDialog.tsx [hunk 3]
```diff
@@ -0,0 +1,133 @@
...
+import { Dialog } from '@material-ui/core';
+import { useSelectChainStyled } from './useSelectChainStyled';
+import { ReactComponent as EthereumIcon } from './assets/ethereum.svg';
+import { ReactComponent as BinanceIcon } from './assets/binance.svg';
+import { ReactComponent as HecoIcon } from './assets/heco.svg';
+import { ModalCloseBtn } from 'modules/uiKit/ModalCloseBtn';
+import { useAccount } from 'modules/account/hooks/useAccount';
+import { t } from 'modules/i18n/utils/intl';
+
+export interface AddEthereumChainParameter {
```
Comment (Denise54321) [inline src/modules/layout/components/Header/components/SelectChainDialog/SelectChainDialog.tsx]: IAddEthereumChain

File: src/modules/layout/components/Header/components/SelectChainDialog/useDialog.tsx [hunk 4]
```diff
@@ -0,0 +1,15 @@
+import { useCallback, useState } from 'react';
+
+export const useDialog = () => {
```
Comment (Denise54321) [inline src/modules/layout/components/Header/components/SelectChainDialog/useDialog.tsx]: useDialogState

File: src/modules/layout/components/Header/Header.tsx [hunk 5]
```diff
@@ -37,10 +39,29 @@ export const Header = () => {
...
+      case 1:
+        return 'Ethereum';
+      case 56:
+        return 'Binance Smart Chain';
+      case 128:
+        return 'Heco Chain';
+      default:
+        return 'Unknown network';
+    }
+  }, [chainId]);
```
Comment (Denise54321) [inline src/modules/layout/components/Header/Header.tsx:70]: Can add 137, Matic network later

File: src/modules/layout/components/Header/components/SelectChainDialog/SelectChainDialog.tsx [hunk 6]
```diff
@@ -0,0 +1,133 @@
...
+    icon: JSX.Element;
+    title: string;
+    subTitle?: string;
+    chainConfig: AddEthereumChainParameter;
+  }) => {
+    return (
+      <div
+        className={classes.cardItem}
+        onClick={() => {
+          handleChangeNetworkToSupported(chainConfig);
```
Comment (Denise54321) [inline src/modules/layout/components/Header/components/SelectChainDialog/SelectChainDialog.tsx]: Don't write method here, make a new method.

**Review by Denise54321 — APPROVED** [review]

**Comment by XuMinhui** [comment]
adding multiple chains

**Pull request merged**